src/main/java/com/alkemy/ong/exception/GlobalControllerExceptionHandler.java

```diff
@@ -56,16 +64,28 @@
 
     }
     }*/
-  
-  @ExceptionHandler(value = {TestimonialNotFoundException.class})
-    protected ResponseEntity<ErrorResponse> handleTestimonialNotFoundException(TestimonialNotFoundException exc){
-  
+
+    @ExceptionHandler(value = {TestimonialNotFoundException.class})
+    protected ResponseEntity<ErrorResponse> handleTestimonialNotFoundException(TestimonialNotFoundException exc) {
+
         ErrorResponse error = new ErrorResponse();
-      
+
         error.setStatus(HttpStatus.NOT_FOUND.value());
         error.setMessage("Testimonial not found");
         error.setTimeStamp(ZonedDateTime.now());
-  
+
         return new ResponseEntity<>(error, HttpStatus.NOT_FOUND);
     }
+
+
+    @ExceptionHandler(value = {BadUserLoginException.class})
+    protected void handleBadUserLoginException(HttpServletResponse response) throws IOException {
+        response.setContentType("application/json;charset=UTF-8");
```

Comment:
Could you please use the constant `MediaType.APPLICATION_JSON_VALUE` instead the hardcoded value.